packages/cockpit/src/transactions/shared/export.js

```diff
@@ -0,0 +1,313 @@
+import {
+  __,
+  always,
+  append,
+  defaultTo,
+  divide,
+  either,
+  head,
+  ifElse,
+  is,
+  isEmpty,
+  isNil,
+  join,
+  juxt,
+  last,
+  length,
+  map,
+  path,
+  pathOr,
+  pathSatisfies,
+  pipe,
+  pick,
+  prop,
+  propEq,
+  props,
+  propIs,
+  propOr,
+  propSatisfies,
+  splitAt,
+  unless,
+  values,
+} from 'ramda'
+import moment from 'moment'
+import acquirerNames from './acquirerNames'
+import cardBrandNames from './cardBrandNames'
+import statusNames from './statusName'
+
+const LIMITER = '-'
+
+const isEmptyOrNill = either(isNil, isEmpty)
+
+const defaultToLimiter = defaultTo(LIMITER)
+
+const propOrLimiter = property => pipe(
+  propOr(LIMITER, property),
+  defaultToLimiter
+)
+
+const isAntifraudScoreNil = propSatisfies(isNil, 'antifraud_score')
+
+const getAntifraudProp = ifElse(
+  isAntifraudScoreNil,
+  always(LIMITER),
+  propOrLimiter('risk_level')
+)
+
+const pickCardDigits = pick(['card_first_digits', 'card_last_digits'])
+
+const concatCardDigits = pipe(
+  pickCardDigits,
+  values,
+  join('******')
+)
+
+const getCardNumber = ifElse(
+  propIs(String, 'card_first_digits'),
+  concatCardDigits,
+  always(LIMITER)
+)
+
+const getCustomerName = pipe(
+  path(['customer', 'name']),
+  defaultToLimiter
+)
+
+const getCustomerSubProp = subProp => pathOr(LIMITER, ['customer', subProp])
+
+const getAddressProp = either(
+  path(['billing', 'address']),
+  path(['address'])
+)
+
+const getAddressSubProp = property =>
```

Comment:
uma sugestão, com o codigo abaixo não precisaria definir a função `getAddressProp`, pois com o `pathOr` dessa forma já acessaria a prop necessária 

```suggestion
const getAddressSubProp = property => either(
  pathOr(LIMITER, ['billing', 'address', property]),
  pathOr(LIMITER, ['address', property ])
)
```